### DISCUSSION TITLE
简悦文档错误修编

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2268] [讨论组]
这里主要记录简悦的各种文档中的 **错别字 / 错误语句** 等内容。

**Comment by gerrampard** [comment]
原文：当遇到如图的**请看**时，则发生了无法设定的问题。
[链接](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%B7%E5%BC%8F?id=%e6%97%a0%e6%b3%95%e6%9b%b4%e6%96%b0%e9%98%85%e8%af%bb%e6%a8%a1%e5%bc%8f%e7%9a%84%ef%bc%9a%e4%b8%bb%e9%a2%98%e3%80%81%e5%ad%97%e4%bd%93%e6%a0%b7%e5%bc%8f%e3%80%81%e5%ad%97%e4%bd%93%e5%a4%a7%e5%b0%8f%e3%80%81%e7%89%88%e9%9d%a2%e5%b8%83%e5%b1%80)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：JSBox 版无法**上上述**方式自动更新，但 JSBox 会提示哪些插件有更新
[链接](http://ksria.com/simpread/docs/#/%E6%9B%B4%E6%96%B0?id=jsbox-%e6%9b%b4%e6%96%b0)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：→ 右键 → 选项 → 高级设定 → 排除列表`，如图： 
[链接](http://ksria.com/simpread/docs/#/FAQ?id=%e6%8e%92%e9%99%a4%e5%88%97%e8%a1%a8)
（排除列表后多了一个不必要的符号）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：注意： Firefox 版下某些插件无法使用，如 与键盘事件有关的情况下均无法使用，但诸如： Lighbox 等**确**不受影响。
[链接](http://ksria.com/simpread/docs/#/%E6%8F%92%E4%BB%B6%E7%B3%BB%E7%BB%9F?id=%e5%9c%ba%e6%99%af)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：插件的类别，**供**有下图所示的内容
[链接](http://ksria.com/simpread/docs/#/%E6%8F%92%E4%BB%B6%E7%BC%96%E5%86%99?id=%e6%96%b0%e5%bb%ba%e6%8f%92%e4%bb%b6%e8%af%b4%e6%98%8e)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：标注系统 是简悦 2.0 最重要的一个功能，借用此功能，可以让简悦真正**你的成为**生产力工具。
[链接](http://ksria.com/simpread/docs/#/%E6%A0%87%E6%B3%A8?id=%e6%8f%8f%e8%bf%b0)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：此功能升级为自动化服务，细节**请看请前往** 自动化。
[链接](http://ksria.com/simpread/docs/#/%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e8%87%aa%e5%8a%a8%e4%bf%9d%e5%ad%98%e5%88%b0%e6%9c%ac%e5%9c%b0)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：在下图输入你的兑换码（L**I**cense）即可自动找回。
[链接](http://ksria.com/simpread/docs/#/%E9%AB%98%E7%BA%A7%E8%B4%A6%E6%88%B7?id=%e6%89%be%e5%9b%9e%e9%ab%98%e7%ba%a7%e8%b4%a6%e6%88%b7)
(大小写)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：在【设备B】上通过 同步配置文件 将【设备A】的数据信息（通过上步的网盘）同步到【设备B】
在【设备C】的操作与 第二步一样。 **4**
[链接](http://ksria.com/simpread/docs/#/%E5%B7%B2%E8%B4%AD%E5%BF%85%E8%AF%BB?id=%e5%a6%82%e4%bd%95%e5%90%8c%e6%ad%a5%e9%ab%98%e7%ba%a7%e8%b4%a6%e6%88%b7%e5%88%b0%e5%a4%9a%e5%8f%b0%e8%ae%be%e5%a4%87)
（多了一个数字4）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：此**为为** 2.0.0 的 Bug，请升级到 2.1.0 即可解决此问题
[链接](http://ksria.com/simpread/docs/#/%E5%B7%B2%E8%B4%AD%E5%BF%85%E8%AF%BB?id=%e5%90%8c%e6%97%b6%e5%bc%80%e5%90%af%e5%a4%9a%e5%8f%b0%e8%ae%be%e5%a4%87%e8%a2%ab%e9%94%81%e7%9a%84%e8%a7%a3%e5%86%b3%e5%8a%9e%e6%b3%95)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：确保你的**简**已经是高级账户。
[链接](http://ksria.com/simpread/docs/#/Sync?id=%e9%aa%8c%e8%af%81)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：通过 简悦 · 同步助手 终于支持了原生的 PDF 输出功能，**只要**分别在：
[链接](http://ksria.com/simpread/docs/#/Sync?id=pdf)
(这个链接锚点无法精准定位文档错误所在处，往下拉，在‘PDF’处）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：借助 pandoc 简悦也支持 原生导出 Epub，在操作上与上面保持**一直**，即：需要分别在 扩展端 与 同步助手 开启此选项。
[链接](http://ksria.com/simpread/docs/#/Sync?id=epub)
(这个链接锚点无法精准定位文档错误所在处，往下拉，在‘Epub’处）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：只要 保证全局可以调用 pandoc 命令，安装完毕后在命令行下输入 pandoc -v **看到后正常的**输出即可。
[链接](http://ksria.com/simpread/docs/#/Sync?id=%e9%aa%8c%e8%af%81%ef%bc%88windows%ef%bc%89)
(语句不通？）
(这个链接锚点无法精准定位文档错误所在处，往下拉，在‘pandoc’ - windows 验证）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文1：简悦 · 同步助手 内置了邮件发送系统。为了 安全性与隐私性，需要使用用户自己的邮件**作为**，所以 需要开启邮箱的 SMTP服务，
原文2：为了简化难度，此版本 仅支持默认选择的邮件服务提供商，暂不支持**字**定制，未来会考虑更多的定制化。
[链接](http://ksria.com/simpread/docs/#/Sync?id=%e9%82%ae%e4%bb%b6%e6%9c%8d%e5%8a%a1)
(这个链接锚点无法精准定位文档错误所在处，往下拉，在‘邮件服务’处）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文1：1.SMTP 服务地址 → 目前仅提供一些常用的邮件服务提供**上**，未来会考虑定制化。
原文2：4. 接**受**的邮件地址
[链接](http://ksria.com/simpread/docs/#/Sync?id=%e5%a6%82%e4%bd%95%e9%85%8d%e7%bd%ae-3)
(这个链接锚点无法精准定位文档错误所在处，往下拉，在‘如何配置’处）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：如：Bear **Carft** FSNotes 等
[链接](http://ksria.com/simpread/docs/#/Textbundle)
（是Craft吧？）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文1: 我本人是一个稍后读的深度使用者，**用户**几乎市面上全部的稍后读，
原文2: 含义：使用此功能后，在进入某个具体的稍后读后，如果此稍后读**在在**本地存在 HTML，则优先加载本地文件。
原文3:随着Zettelkasten 笔记法的流行，我发现标注卡**竟然**与 ZK 卡片**竟然**承载着几乎一致的信息量。
原文4:在导出方面具有**可以**其它剪藏工具不具有的优势。
原文5: 2.2.0 版**再**此基础上加强了这部分，使其实现了，
原文6: 如：Bear **Carft** FSNotes 等
原文7:如果在使用`(时)`出现了问题，请尽量反馈到 这里
[2.2.0更新文档](http://ksria.com/simpread/welcome/version_2.2.0.html)

**Comment by gerrampard** [comment]
原文：加入 稍后读 后，导出**到** HTML 到本地 以及给我 发送邮件
[链接](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%8C%96?id=%e8%83%bd%e5%81%9a%e4%bb%80%e4%b9%88)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：当使用 标注 时，由于 API 限制导致仅支持**的**部分服务
[链接](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%8C%96?id=%e6%b3%a8%e6%84%8f)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：如果你`(是)`一个双链笔记使用者，可以直接 在这里 找到官方定制的一些方案。
[链接](http://ksria.com/simpread/docs/#/%E5%AE%9A%E5%88%B6%E5%8C%96%E5%AF%BC%E5%87%BA?id=%e5%ae%98%e6%96%b9%e6%a8%a1%e6%9d%bf)
(少了个**是**字）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：可更改，通过 选项 → **聚焦模式** → 快捷键 或 进入阅读模式后，侧栏 → 设定 icon → 快捷键 修改；
[链接](http://ksria.com/simpread/docs/#/%E5%BF%AB%E6%8D%B7%E9%94%AE?id=%e9%98%85%e8%af%bb%e6%a8%a1%e5%bc%8f)
（此处应为 阅读模式）

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：https://www.inoreader.com/ https://www.**inoreader**.com/
[链接](http://ksria.com/simpread/docs/#/RSSReader?id=%e6%94%af%e6%8c%81%e6%9c%8d%e5%8a%a1)
（inoreader的墙内地址应该是 https://www.innoreader.com  多一个n)

**Reply by Kenshin** [reply]
Done

**Comment by gerrampard** [comment]
原文：以下的步骤仅 针对新用户，如果你稍微使用过简悦的话，完全可以不**同**看下面的内容。
[链接](https://simpread.pro/api/#/%E5%A6%82%E4%BD%95%E5%BC%80%E9%80%9A)

**Reply by Kenshin** [reply]
Done

**Comment by binsee** [comment]
原文： Params 参数中的 `value: <你的 token>` -> `value: <检索内容>`
https://simpread.pro/api/#/api?id=%e8%af%bb%e5%8f%96-list

**Comment by binsee** [comment]
原文：`返回 阅读列表 的 JSON 结构。` -> `返回 稍后读 导出路径 output 下的文件列表`
https://simpread.pro/api/#/api?id=%e8%af%bb%e5%8f%96-reading

这个是返回 output 目录下的文件列表，并不是返回 `simpread_config.json` 文件内的信息

**Reply by binsee** [reply]
补充：
API中，添加url到简悦稍后读，是通过云端将url信息写入到 `simpread_config.json` 文件内的稍后读列表。并不会在云端抓取页面内容，写入到 `output` 目录下。
API中，`reading` 接口读取的是 `output` 目录下的文件，而 `list` 列举的是 `simpread_config.json` 文件内的稍后读信息。**两者**是有本质不同的，且在设置或使用方式不一致时，很容易会出现两个接口返回信息不一致的情况。

**Comment by gerrampard** [comment]
原文：使得稍后读在当下「信息爆炸」时代成为**一种**对信息负载的一种权宜手段
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e7%a8%8d%e5%90%8e%e8%af%bb%e7%9a%84%e3%80%8c%e9%97%ae%e9%a2%98%e3%80%8d)
（可删除，两个“一种”）

**Comment by gerrampard** [comment]
原文：（希望将这篇文章永久的 **Cpature** 下来，以备不时之需。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e7%a8%8d%e5%90%8e%e8%af%bb%e7%9a%84%e3%80%8c%e9%97%ae%e9%a2%98%e3%80%8d)
(应为：Capture）

**Comment by gerrampard** [comment]
原文：所以如果没有良好的消化机制，就会**越积阅读**，导致积食难消。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e7%a8%8d%e5%90%8e%e5%86%8d%e8%af%bb%ef%bc%9f)
（越积越多）

**Comment by gerrampard** [comment]
原文：因本篇文章并不涉及到此，且少数派这边有多篇这样的文章，就不**在**引申。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e8%a7%a3%e6%9e%84)
（在-》再）

**Comment by gerrampard** [comment]
原文：而且在同步速度上**跟**传统稍后读产品无法比拟。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e6%9b%b4%e5%a5%bd%e7%9a%84-capture-%e6%96%b9%e6%a1%88)
（‘跟’字删除）

**Comment by gerrampard** [comment]
原文：而作为信息最好的载体：稍后读（包含 Capture），天然**对**更适合使用此方式
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e5%8f%8c%e5%90%91%e9%93%be%e6%8e%a5)
(对 -> 地)

**Comment by gerrampard** [comment]
原文：看起来跟其它双链笔记**一页**，很酷炫，仅此而已？
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e7%9f%a5%e8%af%86%e5%9b%be%e8%b0%b1)

**Comment by gerrampard** [comment]
原文：当鼠标随便指向某个信息点后，就会产生**于**此关联的信息。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e7%9f%a5%e8%af%86%e5%9b%be%e8%b0%b1)
（于 -》与）

**Comment by gerrampard** [comment]
原文：信息**的**之间的关联性联系
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%80%E4%B8%AA%E6%9B%B4%E5%A5%BD%E7%9A%84%E7%A8%8D%E5%90%8E%E8%AF%BB?id=%e8%ae%a9%e4%bf%a1%e6%81%af%e6%9b%b4%e5%a5%bd%e7%9a%84%e6%9c%8d%e5%8a%a1%e4%ba%8e%e4%bd%a0)
（‘的’ 重复）

**Comment by gerrampard** [comment]
原文：如果有一天简悦不**在**开发了，如何能保障简悦的高级账户一直能使用下去？
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E7%9A%84%E8%BD%BB%E5%90%8E%E7%AB%AF%E6%A8%A1%E5%BC%8F?id=%e8%bd%bb%e5%90%8e%e7%ab%af%e6%a8%a1%e5%bc%8f)
（在-》再）

**Comment by gerrampard** [comment]
原文：并不是每个服务`都`需要把数据保存在本地，
[链接](http://ksria.com/simpread/wiki/#/%E7%94%A8%E6%88%B7%E6%8B%A5%E6%9C%89%E5%AE%8C%E6%95%B4%E7%9A%84%E6%95%B0%E6%8D%AE%E6%89%80%E6%9C%89%E6%9D%83?id=%e9%9a%90%e7%a7%81%e6%9d%83%e8%bf%98%e6%98%af%e6%98%93%e7%94%a8%e6%80%a7%ef%bc%9f)　
少了‘都’

**Comment by gerrampard** [comment]
原文：无需多言，自建服务肯定远**生**于第三方同步服务。
[链接](http://ksria.com/simpread/wiki/#/%E4%B8%BA%E4%BB%80%E4%B9%88%E4%B8%8D%E8%87%AA%E5%BB%BA%E5%90%8C%E6%AD%A5%E6%9C%8D%E5%8A%A1?id=%e6%98%93%e7%94%a8%e6%80%a7)
（生-》胜）

**Comment by gerrampard** [comment]
原文：完全不依赖于服务器，只要 **Chro me** Extensions API 不取消
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E5%9B%9B%E5%91%A8%E5%B9%B4?id=%e7%ae%80%e6%82%a6-20-%e6%98%af%e4%b8%aa%e5%bc%82%e7%b1%bb)
（chrome多了空格）

**Comment by gerrampard** [comment]
原文：Bug 小能手 Yuanio，发现了很多我完全无法**重新**的 Bug
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E5%9B%9B%E5%91%A8%E5%B9%B4?id=%e6%9c%8b%e5%8f%8b%e4%bb%ac)
（重新-》重现）

**Comment by gerrampard** [comment]
原文：你将拥有真正意义/完整**是**数据所有权
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E5%9B%9B%E5%91%A8%E5%B9%B4?id=%e6%9c%80%e5%90%8e)
（是-》的）

**Comment by gerrampard** [comment]
原文：希望做一些让这个世界变得更美好的小事。 _正如我在推上的 bio：_
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E5%9B%9B%E5%91%A8%E5%B9%B4?id=%e6%9c%80%e5%90%8e)
(重复了)

**Comment by gerrampard** [comment]
原文：同时你在 Github Issues 上面的任意提问**都我会**一一回复。
[链接](http://ksria.com/simpread/wiki/#/%E7%AE%80%E6%82%A6%E7%A4%BE%E5%8C%BA?id=%e6%8f%90%e9%97%ae%e7%b3%bb%e7%bb%9f)
（都我会-》我都会）

**Comment by fordchao** [comment]
原文：简悦·同步助手2，导出，倒数第二行：Readwise.io 是一个邮件发送服务，它可以导入多种服务的标记并**已**邮件的形式发给你。
无链接。
（已-》以）

**Comment by lyserenity** [comment]
原文：又成为 反向链接，此功能属于 简悦 · 稍后读 的功能，与其它双链笔记一样，可以方便的将不同的稍后读或标注产生关联。
[链接](http://ksria.com/simpread/docs/#/%E5%8F%8C%E5%90%91%E9%93%BE%E6%8E%A5?id=%e6%8f%8f%e8%bf%b0)
（又成为 -> 又称为）

**Reply by Kenshin** [reply]
Done

**Comment by lyserenity** [comment]
[链接](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%B7%E5%BC%8F?id=class-%e8%af%b4%e6%98%8e)

`.simpread-font` 的 code block 最后疑似少了个 `}`

**Reply by Kenshin** [reply]
Done

**Comment by binsee** [comment]
[URL编辑器](http://ksria.com/simpread/docs/#/URL%E7%BC%96%E8%BE%91%E5%99%A8)

- 白名单
- 排除列表
- 黑名单
> 支持正则表达式，e.g. [[/https:\\/\\/movie.douban.com\\/subject\\/\\d+\\/?/]]
中的正则不应将 `\` 转义为 `\\`

来自 #2539

**Comment by lyserenity** [comment]
![image](https://user-images.githubusercontent.com/32646062/138699941-67c70adb-7069-461d-84d7-764370e071b8.png)  
「请误」更改这里 -> 「请勿」

**Comment by Kenshin** [comment]
![图片](https://user-images.githubusercontent.com/16938012/156759616-b64c4c93-c2fb-442d-820c-529e84f7321f.png)

via @Kukmoon origin https://github.com/Kenshin/simpread/issues/3481
